src/components/activities/ActivityCard.jsx

```diff
@@ -96,7 +98,7 @@ class ActivityCard extends Component {
     super(props);
     this.state = {
       isActivityChecked: false,
-      statuses: ['pending', 'rejected', 'approved', 'in review'],
+      statuses: ['pending', 'rejected', 'approved', 'in review', 'completed'],
```

Comment:
What do you think about having this as a map somewhere outside this component that we can reuse when necessary?

Also, what are `needButtons`?